common/tools/dev-tool/src/config/rollup.base.config.ts

```diff
@@ -141,12 +141,23 @@ export function sourcemaps() {
         const code = await readFile(id, "utf8");
         if (code.includes("sourceMappingURL")) {
           const basePath = path.dirname(id);
-          const mapPath = code.match(/sourceMappingURL=(.*)/)?.[1];
-          if (!mapPath) {
-            this.warn({ message: "Could not find map path in file " + id, id });
+          const mapping = code.match(/sourceMappingURL=(.*)/)?.[1];
+          if (!mapping) {
+            this.warn({ message: "Could not find source mapping in file " + id, id });
             return null;
           }
-          const absoluteMapPath = path.join(basePath, mapPath);
+          if (mapping.startsWith("data:")) {
+            debug("inline source map in", id);
+            if (mapping.startsWith("data:application/json;charset=utf-8;base64,")) {
+              const base64Encoded = mapping.split(",")?.[1];
+              const decoded = Buffer.from(base64Encoded, "base64").toString("utf-8");
+              const map = JSON.parse(decoded);
```

Comment:
should we try/catch the JSON.parse to be safe?

Reply:
I would trust the map files.  Also rollup will catch unhandled error and report to us in the rare case of bad map files.

Reply:
oh right, there's a catch below (line 167) to handle error cases